### PR TITLE
Add a Jenkins job which builds an ISO when a GitHub pull request is created

### PR DESCRIPTION
**Pull request by alanoe**

### CODE REVIEW

File: jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    downstreamParameterized {
+      trigger('build_host_os_iso') {
+	block {
+	  buildStepFailure('FAILURE')
+	  failure('FAILURE')
+	  unstable('UNSTABLE')
+	}
+	parameters {
+	  predefinedProps([BUILDS_REPO_COMMIT: '$sha1',
+			   VERSIONS_REPO_COMMIT: 'master'])
```
Comment (ghost) [inline jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy:40]: This will not work, the build_host_os_iso job needs a build_host_os job ID

**Review by ghost — CHANGES_REQUESTED** [review]

File: jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  properties {
+    githubProjectUrl("https://github.com/${GITHUB_ORGANIZATION_NAME}/builds/")
+  }
+  triggers {
+    githubPullRequest {
+      userWhitelist("${GHPRB_ADMIN_USER}")
+      orgWhitelist("${GHPRB_ADMIN_ORGANIZATION}")
+      allowMembersOfWhitelistedOrgsAsAdmin()
+      cron('H/5 * * * *')
+      triggerPhrase('.*start\\W+(tests|build_iso).*')
```
Comment (farosas) [inline jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy:22]: I used `triggerPhrase('.*start\\W+iso.*')` in the 'trigger_host_os_iso_from_versions' trigger. Also, the triggers for 'build_host_os' all use ` triggerPhrase('.*start\\W+(tests|build).*')`. That '.*' there in the end would make this job trigger a second 'build_host_os'

File: jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      extensions {
+	commitStatus {
+	  context('Build Host OS')
```
Comment (farosas) [inline jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy:25]: Build Host OS ISO

Comment (farosas) [reply jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy:25]: Add
`statusUrl('${JENKINS_URL}/job/build_host_os_iso/${TRIGGERED_BUILD_NUMBER_build_host_os_iso}')`
like the _from_versions trigger

File: jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy (hunk 1 above)
Comment (farosas) [reply jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy:40]: Need to add the script from the other trigger:

https://github.com/open-power-host-os/infrastructure/blob/master/jenkins_jobs/trigger_host_os_iso_build_from_versions_repo.groovy#L36

And the credentials binding plugin:

https://github.com/open-power-host-os/infrastructure/blob/master/jenkins_jobs/trigger_host_os_iso_build_from_versions_repo.groovy#L53

**Review by farosas — CHANGES_REQUESTED** [review]

**Comment by olavph** [comment]
This is superseded by #77

**Pull request closed without merging**